src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,52 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.file.reader.FileReader;
+import core.basesyntax.file.reader.FileReaderImpl;
+import core.basesyntax.file.writer.FileWriter;
+import core.basesyntax.file.writer.FileWriterImpl;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.handler.impl.BalanceOperationHandler;
+import core.basesyntax.handler.impl.PurchaseOperationHandler;
+import core.basesyntax.handler.impl.ReturnOperationHandler;
+import core.basesyntax.handler.impl.SupplyOperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.report.ReportGenerator;
+import core.basesyntax.report.ReportGeneratorImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class HelloWorld {
-    // HINT: In the `public static void main(String[] args)` it is better to create instances of your classes, 
-    // and call their methods, but do not write any business logic in the `main` method!
+    public static void main(String[] args) {
```

Comment:
Don't keep all logic in a single package. Your project structure should consist of different packages such as `db`, `model`, `service`, `service.impl`, `strategy`, etc. It seems that the 'HelloWorld' class is not correctly named according to its responsibilities and it's in the default package. Consider placing it in a suitably named package and renaming it to reflect its role in the application, such as 'MainApplication' or similar.